packages/kbn-optimizer/limits.yml

```diff
@@ -75,7 +75,7 @@ pageLoadAssetSize:
   grokdebugger: 5484
   guidedOnboarding: 26999
   home: 13710
-  imageEmbeddable: 6000
```

Comment:
The imageEmbeddable was failing the limits check because it was over by only a few bytes. I updated this value using `node scripts/build_kibana_platform_plugins --update-limits`.